### PR TITLE
Run kotlin linting during CI

### PR DESCRIPTION
**Pull request by mkalish**
This PR updates CI to lint the kotlin files.  Occasionally a linting error sneaks in and this prevents it.

Test Steps:
1. n/a

## Changes
- Adds the `ktlintCheck` to the CI build

### Testing
- [ ] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

### Process
- [ ] Are there licensing issues with any new dependencies introduced?
- [ ] Includes a summary of what a code reviewer should test/verify?
- [ ] Updated the release notes?
- [ ] Database changes are submitted as a separate PR?
- [ ] DevOps team has been notified if PR requires ops support?

### CODE REVIEW

**Review by jimmyfagan — APPROVED** [review]
LGTM!

File: prime-router/src/main/kotlin/fhirengine/engine/FHIREngine.kt [hunk 1]
```diff
@@ -152,7 +152,8 @@ abstract class FHIREngine(
 
     /**
      *
-     * Responsible for invoking the [doWork] function, inserting any new tasks and updating the previous task
+     * Responsible for invoking the [doWork] function, inserting any new tasks and updating the previous task and
```
Comment (jack-h-wang) [inline prime-router/src/main/kotlin/fhirengine/engine/FHIREngine.kt:155]:
```suggestion
     * Responsible for invoking the [doWork] function, inserting any new tasks, updating the previous task and
```

**Review by jack-h-wang — APPROVED** [review]
Just a minor suggested change, otherwise LGTM!

**Review by jalbinson — APPROVED** [review]

**Review by JFU-NAVA-PBC — APPROVED** [review]
LGTM

**Pull request merged**